README.md

```diff
@@ -38,21 +40,21 @@ $ fun -h
     build               Build the dependencies
 ```
 
-## 使用
+## Usage
 
-安装完命令行工具之后，即可开始进行代码的开发了。为了配合 fun 工具，您需要创建一个工程目录，然后在工程目录下创建一个 `template.yml` 模板文件，fun 会将该目录视为一个工程。
+Once you have installed the command line tools, you are ready to write code. In order to use fun tool, you need to create a project directory and then create a file named template.yml in the project directory. Fun will treat the directory as a project.
 
-我们将在该模板文件中定义项目相关的信息。fun 可以定义的信息参见 [fun 的规范文档](https://github.com/aliyun/fun/blob/spec_for_ros/docs/specs/2018-04-03-zh-cn.md)。
+We will define a series of resources in this template file. Resources that can be defined by fun can be found on [fun's specification document](https://github.com/aliyun/fun/blob/spec_for_ros/docs/specs/2018-04-03.md).
 
-在使用前，我们需要先进行配置，通过键入 `fun config`，然后按照提示，依次配置 `Account ID`、`Access Key Id`、`Secret Access Key`、 `Default Region Name` 即可。
+However before using `fun`, we need to configure it first by typing `fun config` and then following the prompts, configure `Account ID`, `Access Key Id`, `Secret Access Key` and `Default Region Name`.
 
-完成 config 操作后，fun 会将配置保存到用户目录下的 `.fcli/config.yaml` 文件中。
+After the `fun config` is completed, fun saves the configuration to the `.fcli/config.yaml` file in the user home.
 
-配置完成后，就可以使用 fun 命令了，模板文件、代码编写完成后，通过 fun deploy 就可以一键将服务部署到线上环境了。
+Now you are ready to use the fun command. 
 
-## 示例
+### Example
 
-下面我们一个示例来演示 fun 如何使用。首先在项目根目录下创建一个 hello.js 文件。
+Here is an example to demonstrate how fun is used. First create a hello.js file in the project root directory.
```

Comment:
Here is an example. First create a hello.js file in the project root directory.

Reply:
done